.github/pull_request_template.md

```diff
@@ -1,80 +1,35 @@
+Main description of what's done here, shortly, with reasoning, references, thoughts and ideas for better understanding.
+This text is intended to be removed and replaced with actual description. Other items can also be removed or edited if not relevant. If any of following items require more description, it should also go in this block.
+Please, read all items and make an action every time to create more conscious PR. Thanks!
```

Comment:
```suggestion
Please, read all items and make an action every time to create more consistent pull requests. Thanks!
```

Reply:
My idea is to remove non relevant sections, that's why I use "action" here, developer can edit, remove, fill in or just tick a checkbox. Does it make sense?